backstop.config.js

```diff
@@ -15,6 +15,8 @@ const scenarios = findAllPages()
   .filter(pageName => pageName.indexOf('permutations') !== -1 && skippedTests.indexOf(pageName) === -1)
   .map(pageName => ({
     label: pageName,
+    // wait until code-editor is fully loaded and rendered error state
+    readySelector: pageName === 'code-editor/permutations' ? '.ace_error' : undefined,
```

Comment:
[minor] could be nice to move this out from inline and have it defined in an extensible way next to `skippedTests` (e.g. `const readySelectors = {'code-editor/permutations': '.ace_error'}` or similar), so that all page-specific configuration is in one place

Reply:
Agree, but this can wait for a second condition in this list (not sure if this ever happen)